### ISSUE TITLE
Custom keybindings removed by other Minecraft versions

### ISSUE THREAD

**Issue by 61max** [HGLabor/client-issues#189]
Moin liebes NoRisk Client Team,
ich habe einen Bug gefunden.
Ich will meine "Freelook Prespective" nicht auf der Taste R haben, deswegen habe ich sie in den Tasteneinstellungen von Minecraft auf P gebindet. Allerdings immer wenn ich NoRisk Client schließe und wieder neu starte, resettet sich der Keybind wieder auf R.
Wäre nett, wenn ihr das fixen könntet.
LG
Max

**Comment by KeineLustAufName** [comment]
Öffnest du vllt zwischendurch eine andere mc Version?

**Comment by 61max** [comment]
ja, meistens badlion client oder labymod aber auch in der 1.8.9

**Comment by KeineLustAufName** [comment]
Ja liegt wahrscheinlich daran, ich hab das auch. Aber das passiert auch in der 'normalen' mc Version

**Comment by F0Xde** [comment]
Stimmt, vielleicht sollten wir unsere eigenen Keybinds separat speichern.
